auth0/src/main/java/com/auth0/android/request/DefaultClient.kt

```diff
@@ -40,7 +40,8 @@ public class DefaultClient() : NetworkingClient {
         when (options.method) {
             is HttpMethod.GET -> {
                 // add parameters as query
-                options.parameters.map { urlBuilder.addQueryParameter(it.key, it.value) }
+                options.parameters.filterValues { it is String }
```

Comment:
when building the query, OKHttp requires us to pass `String?` type for the values. I'm filtering those that are not a String. Since the Request interface requires `String, String` this should not be an issue for Requests built by our API clients.